nss_cache/sources/httpsource.py

```diff
@@ -36,6 +36,8 @@
 from nss_cache.util import file_formats
 from nss_cache.util import curl
 
+from io import StringIO
```

Comment:
Please put the std headers above the local; this to line 22, like you did in the other files.